neqo-transport/src/cc/tests/new_reno.rs

```diff
@@ -129,3 +129,83 @@ fn issue_876() {
     cwnd_is_halved(&cc);
     assert_eq!(cc.bytes_in_flight(), 4 * MAX_DATAGRAM_SIZE);
 }
+
+#[test]
+// https://github.com/mozilla/neqo/pull/1465
+fn issue_1465() {
+    let mut cc = ClassicCongestionControl::new(NewReno::default());
+    let mut pn = 0;
+    let mut now = now();
+    let next_packet = |pn: &mut PacketNumber, now| {
+        let p = SentPacket::new(
+            PacketType::Short,
+            *pn,               // pn
+            now,               // time_sent
+            true,              // ack eliciting
+            Vec::new(),        // tokens
+            MAX_DATAGRAM_SIZE, // size
+        );
+        *pn += 1;
+        p
+    };
+    let send_next = |cc: &mut ClassicCongestionControl<NewReno>, pn: &mut PacketNumber, now| {
```

Comment:
Just curious, but I thought that you could close over the packet number parameter safely (because it would only be used within the closure.  Was that not the case?

Reply:
Tbh, I didn't try, because I didn't notice that this could work. I just repeated the same thing as with `cc`.